lib/public_key/src/public_key.erl

```diff
@@ -2238,7 +2238,13 @@ cacerts_get() ->
 %%--------------------------------------------------------------------
 -doc(#{title => <<"Certificate API">>,
        since => <<"OTP 25.0">>}).
--doc "Loads the OS supplied trusted CA certificates.".
+-doc """
```

Comment:
If I may... I'd add an example on how to actually do that (e.g. via command line to `erl`, also mentioning that you can inject an OS environment variable there - `erl -public_key cacerts_path ${CACERTS_PATH}`. While I understand it could be obvious for long-time Erlang users, it may puzzle first timers.

Reply:
More docs and examples is the right call, pushed!